kedro-org/kedro-viz
# Upgrade to React v17 and MUI v5

## Description

Resolves #1265.

## Development notes

I've gone up one major version, to React 17. I also bumped Google's Material UI component library to v5 from v4, since we'll need to do that when we upgrade to React 18.

## QA notes

Please delete both the `node_modules` folder and the `package-lock.json` file. Then run `npm install` and make sure everything installs and runs without issue.

## Checklist
